src/lib/translators/mongo4.ts

```diff
@@ -30,5 +30,7 @@ function transformConvert(step: Readonly<ConvertStep>): MongoStep {
   return { $addFields: mongoAddFields };
 }
 
-export class Mongo40Translator extends Mongo36Translator {}
+export class Mongo40Translator extends Mongo36Translator {
+  label = 'Mongo 4.0';
```

Comment:
NOTE: this will be coiped on all instances, I had imagined something like `static label = 'Mongo 4.0'` but then you would probably need to write `myTranslator.constructor.label` to access it from the instance

Reply:
I agree this is better ! https://github.com/ToucanToco/weaverbird/pull/406/files#diff-3480543ec07dddab995da489925679aeR72